homeassistant/generated/config_flows.py

```diff
@@ -73,6 +73,7 @@
     "monoprice",
     "mqtt",
     "myq",
+    "mystrom_button",
```

Comment:
This looks unrelated.

Reply:
It's is. Looks like a left-over in the filesystem.

Reply:
```suggestion
```